### ISSUE TITLE
(typescript/member-delimiter-style) produces invalid code

### ISSUE THREAD

**Issue by trusktr** [bradzacher/eslint-plugin-typescript#148]
When using the `"delimiter": "none"` option, it will change

```ts
	rendererSize: { x: number, y: number }
```

into 

```ts
	rendererSize: { x: number y: number }
```

which is invalid.

The expected is for it to become:

```ts
	rendererSize: {
		x: number
		y: number
	}
```

or something valid.

**Comment by mightyiam** [comment]
I don't feel that it should introduce line breaks. What I would like is #92.

**Comment by mightyiam** [comment]
And then, it should fix according to the single-line setting.

**Comment by bradzacher** [comment]
duplicate of #92
